.gitignore

```diff
@@ -81,8 +81,6 @@ htmlcov/
 
 *errorShots*
 
-data/cy
```

Comment:
Why? Our Welsh translated schemas gets put in here. This risks a poorly translated schema getting into master.

Reply:
We need this for the `test_language` schema that's use in a functional else. But perhaps it's worth keeping the ignore in place and just forcing that file in as an exception?

Reply:
Will the properly translated schemas not be put here (assuming we've not switched to a registry by then)?  For now the only welsh schema is of `test_language`, which really helps with the testing of the new functionality and doesn't need a perfect translation.

Reply:
The properly translated schemas will be put here. At the moment however we don't have any properly translated schemas. We do however have some scripts that will create some very poorly translated census schemas that may end up getting into master if we take this ignore out.

Reply:
`translate_schemas.sh` isn't working very well right now.  There's a card to fix it so until then I'll leave `cy/test_language.json` committed for the benefit of the tests and keep the line in `.gitignore`.